modules/msg/CMakeLists.txt

```diff
@@ -0,0 +1,62 @@
+##################################################################
+#
+# cFE message module CMake build recipe
+#
+# This CMakeLists.txt adds source files for
+# message module included in the cFE distribution.  Selected
+# files are built into a static library that in turn
+# is linked into the final executable.
+#
+# Note this is different than applications which are dynamically
+# linked to support runtime loading.  The core applications all
+# use static linkage.
+#
+##################################################################
+
+# Add the basic set of files which are always built
+# Defined as absolute so this list can also be used to build unit tests
+set(${DEP}_SRC
```

Comment:
I only see one suggestion which I'm fine with (document), but it seems like you had an alternative in mind?

Reply:
The alternative would be to use a fixed name (`msg`) and not make it variable.  Is there an expectation that this will be compiled under a different library name?  A compromise might be to set your own variable name at the beginning, and use that variable for the rest of the script.  Depends on the expected use-case for compiling the same library under a different name.

Reply:
I'd think hard coding it would be a less desirable solution.  The name should match the module name such that the system can utilize common patterns.  I'd rather avoid setting up a potential for the name to not match.